Sources/TensorFlow/Operators/Basic.swift

```diff
@@ -378,6 +378,37 @@ extension Tensor {
     return transposed(permutation: Tensor<Int32>(defaultPermutations))
   }
 
+  /// Returns a tensor with specified dimensions reversed.
+  /// - Precondition: Each value in `axes` must be in the range `-rank..<rank`.
+  /// - Precondition: There must be no duplication in `axes`.
+  @inlinable
+  @differentiable(wrt: self where Scalar: TensorFlowFloatingPoint)
+  func reversed(inAxes axes: Tensor<Int32>) -> Tensor {
+    precondition(areAxesInRange(axes), "All axes must be in the range `[-rank, rank)`.")
+    return _Raw.reverseV2(self, axis: axes)
+  }
+
+  /// Returns a tensor with specified dimensions reversed.
+  /// - Precondition: Each value in `axes` must be in the range `-rank..<rank`.
+  /// - Precondition: There must be no duplication in `axes`.
+  @inlinable
+  @differentiable(wrt: self where Scalar: TensorFlowFloatingPoint)
+  func reversed(inAxes axes: [Int]) -> Tensor {
+      precondition(axes.count == Set(axes.map { $0 < 0 ? $0 + rank : $0 }).count,
+                   "There must be no duplication in axes.")
+      let axes = axes.map(Int32.init)
+      return reversed(inAxes: Tensor<Int32>(axes))
```

Comment:
I ran `swift-format` but some lines are not 2-space indented.
`swift-format --configuration .swift-format Sources/TensorFlow/Operators/Basic.swift -i`
What's happening?

Reply:
I wouldn't worry too much about the swift-format right now. I would prefer it to be some sort of presubmit check in which case there is no ambiguity about whose installation is official or not. When that happens, we'll have to re-run it.

Reply:
> `swift-format --configuration .swift-format Sources/TensorFlow/Operators/Basic.swift -i`

Note: if you run `swift-format` anywhere in the `swift-apis` directory (the top-level contains the `.swift-format` configuration file), there's no need to pass `--configuration`. `swift-format` can find it, looking recursively up the directory structure.

From https://github.com/apple/swift-format#configuration:
> For any source file being checked or formatted, `swift-format` looks for a JSON-formatted file named `.swift-format` in the same directory. If one is found, then that file is loaded to determine the tool's configuration. If the file is not found, then it looks in the parent directory, and so on.